website/docs/api/language.md

```diff
@@ -827,13 +828,13 @@ subclass of the built-in `dict`. It supports the additional methods `to_disk`
 
 ## Language.to_disk {#to_disk tag="method" new="2"}
 
-Save the current state to a directory. If a model is loaded, this will **include
-the model**.
+Save the current state to a directory. If a trained pipeline is loaded, this
+will **include all model data**.
```

Comment:
I think it's not clear what "model" is here. All data for each model in each component of the pipeline?

Reply:
Maybe something more like "If a trained pipeline is loaded, this will **include all data** for each component, including the trained weights of the internal models" ?

Reply:
Good point! I just edited it and also added something about how `Language.to_disk` delegates to the component method, if available.